audio.py

```diff
@@ -6,182 +6,182 @@
 import scipy.io.wavfile
 
 smax = torch.iinfo(torch.int16).max
-f2s_numpy = lambda signal, max=np.float32(smax): np.multiply(signal, max).astype('int16')
-s2f_numpy = lambda signal, max=np.float32(smax): np.divide(signal, max, dtype='float32')
+f2s_numpy = lambda signal, max = np.float32(smax): np.multiply(signal, max).astype('int16')
+s2f_numpy = lambda signal, max = np.float32(smax): np.divide(signal, max, dtype = 'float32')
```

Comment:
This file only contains codestyle fixes.
Last Alexander Abugaliev commit in master did not follow the repository codestyle rules, which lead to merge conflicts.